core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6259,6 +6259,37 @@ private void checkLiteral2(String expression, String expected) {
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBiqQuery));
   }
 
+  @Test public void testRegexpInstr() {
+    final RelBuilder builder = relBuilder();
+    final RexNode regexpInstr2Args = builder.call(SqlLibraryOperators.REGEXP_INSTR,
```

Comment:
We can have more meaningful name here. Suggestion

regExpInstrWithTwoArgs

Please change at other places as well

Reply:
Done